hr_payroll_run_report_xls/wizard/print_run_report.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data>
+
+		<!-- Wizard view -->
+		<record id="print_run_report_form" model="ir.ui.view">
+			<field name="name">print.run.report.form</field>
+			<field name="model">print.run.report</field>
+			<field name="arch" type="xml">
+				<form string="Print Run Report">
+					<footer>
+						<button name="print_report" string="Print Register" type="object"
+							context="{'xls_export':1}" default_focus="1" class="oe_highlight" />
+						or
+						<button string="Cancel" class="oe_link" special="cancel" />
+					</footer>
+				</form>
+			</field>
+		</record>
```

Comment:
can you put indentation to 4 spaces or 2 spaces please. This is a bit unreadable.